subprojects/test-runner/report-viewer/src/main/kotlin/com/avito/report/ReportsApiFactory.kt

```diff
@@ -46,7 +37,7 @@ public object ReportsApiFactory {
                         }
                     }
                     .build(),
-                gson = gson
+                gson = createReportGson()
```

Comment:
reuse single fun; it's still a mess, because some entities are used between `repor-api` and `report-viewer`